modules/sagemaker/sagemaker-templates-service-catalog/README.md

```diff
@@ -153,6 +166,8 @@ parameters:
         name: studio
         key: StudioDomainArn
 ```
+### Sample manifest example with GitHub as source repositiry
```

Comment:
nit - typo. 

Also few notes about the readme:
- Above GitHub inputs are in required section although they are optional except `repository-type`
- Would be great to provide both manifest examples for default (CodeCommit) and GitHub in the readme and clearly state these are the two options when using the module. Prerequisites section leads me to believe GitHub is the only option

Reply:
Updated README and also added manifest for CodeCommit repo configuration